resources/i18n/en.json

```diff
@@ -14,6 +16,7 @@
     "print":"Print",
     "new-tab":"New tab",
     "close-tab":"Close tab",
+    "close-dialog":"Close",
```

Comment:
Why not call this simply "close" or "close-button" ?
This is not specific to closing a dialog. This is more about the action of closing something.

Reply:
@HanYoung-uwu any feedback here?

Reply:
done